core/project.clj

```diff
@@ -60,4 +60,5 @@
                                          ; more recent version provided by shadow-cljs
                                          :exclusions [com.google.javascript/closure-compiler-unshaded]]
                                         [thheller/shadow-cljs "2.20.14"]]
-                         :source-paths ["src/main/cljs" "src/main/cljc" "src/test/cljs" "src/test/cljc"]}})
+                         :source-paths ["src/main/cljs" "src/main/cljc" "src/test/cljs" "src/test/cljc"]}
+             :uberjar {:uberjar-exclusions [#"org/slf4j"]}})
```

Comment:
This is a transitive dependency from "org.bitbucket.b_c/jose4j", which causes an issue with my demoproject. With this exclusion, I can at least build and run my demo project successfully, but there's a huge amount of debug log noise coming from Jarl, which I haven't been able to change yet.